app/models/user.py

```diff
@@ -77,7 +78,7 @@ class User(SoftDeletionModel):
     is_verified = db.Column(db.Boolean, default=False)
     was_registered_with_order = db.Column(db.Boolean, default=False)
     last_accessed_at = db.Column(db.DateTime(timezone=True))
-    created_at = db.Column(db.DateTime(timezone=True), default=datetime.now(pytz.utc))
+    created_at = db.Column(db.DateTime(timezone=True), default=func.now())
```

Comment:
Wow. If I understand it correctly, this is really bad. Because it will always use the same value for created at, which will be the datetime when the server started and this value got initialised. Please confirm if that is the case.

Is this present everywhere else as well

Reply:
@iamareebjamal No, this is not the case for sure. I tested it several times myself. Plus, this is not what this function does at all. It's used when you want a default timestamp with (timezone=true). Other way was pytz, but it was showing that absurd behaviour.

Reply:
I'm saying that it was doing this behaviour previously. Please revert and confirm that each time created_at will be same and only change when you restart the server

Reply:
@iamareebjamal This is not true because last_accessed_at uses the same old function, and it's working fine.

Reply:
It's not using the same function `last_accessed_at = db.Column(db.DateTime(timezone=True))`

Reply:
@iamareebjamal So, utcnow is saving timestamp adding 1 hour.(https://stackoverflow.com/questions/28573873/python-datetime-utcnow-returning-incorrect-datetime) I think it should be replaced there as well.

Reply:
Replace where? Where else is it used?

Reply:
@iamareebjamal Used in session table as well. I am looking into other places as well.

Reply:
Replace all places where default is set to datetime.now

Reply:
@iamareebjamal Ok. That's all.